test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -274,38 +273,97 @@ public void Should_Write_History_For_Audited_Entities_Update()
         }
 
         [Fact]
-        public void Should_Write_History_For_Audited_Entities_Update_Owned()
+        public void Should_Write_History_For_Owned_Entity_Of_Audited_Entities_Create()
         {
-            /* Blog has Audited attribute. */
+            // Blog is the owner of BlogEx and has Audited attribute.
+            // Therefore, BlogEx should follow Blog and have entity history.
 
-            int blog1Id;
-            var newValue = "blogger-2";
-            string originalValue;
+            WithUnitOfWork(() =>
+            {
+                // since owned entities is not available via DbContext -> DbSet,
+                // we have to load it via it's owner entity
+                var blog1 = _blogRepository.Single(b => b.Name == "test-blog-2");
+                blog1.More.ShouldBeNull();
+            });
 
-            using (var uow = Resolve<IUnitOfWorkManager>().Begin())
+            WithUnitOfWork(() =>
             {
+                var blog1 = _blogRepository.Single(b => b.Name == "test-blog-2");
```

Comment:
It's unnecessary to split this from the block above.

```diff
  WithUnitOfWork(() =>
  {
      var blog2 = _blogRepository.Single(b => b.Name == "test-blog-2");
      blog2.More.ShouldBeNull();
- });
-
- WithUnitOfWork(() =>
- {
-     var blog2 = _blogRepository.Single(b => b.Name == "test-blog-2");
      blog2.More = new BlogEx() { BloggerName = "blogger-2" };
  });
```

Reply:
You’re right, probably due to copy and paste